<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBoxPorts"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSelect">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonClose">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widgetSerial" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Flow control</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="checkBoxDtr">
            <property name="text">
             <string>DTR</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelBaudRateMain">
            <property name="text">
             <string>Baud rate in</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBoxBaudRateRead">
            <property name="maximum">
             <number>999999</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBoxBreakEn">
            <property name="text">
             <string>Break enable</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Stop bits</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Parity</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBoxFlowControl"/>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="comboBoxParity"/>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="checkBoxRts">
            <property name="text">
             <string>RTS</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QComboBox" name="comboBoxStopBits"/>
          </item>
          <item row="2" column="5">
           <widget class="QPushButton" name="pushButtonSet">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Set</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QSpinBox" name="spinBoxDataBits">
            <property name="maximum">
             <number>999999</number>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Data bits</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="spinBoxBaudRateWrite">
            <property name="maximum">
             <number>999999</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="labelBaudRateSecond">
            <property name="text">
             <string>Baud rate out</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEditText"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonSendText">
           <property name="text">
            <string>Send text</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="lineEditHex"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonSendHex">
           <property name="text">
            <string>Send HEX</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCustom">
         <property name="text">
          <string>Custom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditLog">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>comboBoxPorts</tabstop>
  <tabstop>pushButtonSelect</tabstop>
  <tabstop>pushButtonClose</tabstop>
  <tabstop>spinBoxBaudRateRead</tabstop>
  <tabstop>spinBoxBaudRateWrite</tabstop>
  <tabstop>spinBoxDataBits</tabstop>
  <tabstop>comboBoxFlowControl</tabstop>
  <tabstop>comboBoxParity</tabstop>
  <tabstop>comboBoxStopBits</tabstop>
  <tabstop>checkBoxBreakEn</tabstop>
  <tabstop>checkBoxDtr</tabstop>
  <tabstop>checkBoxRts</tabstop>
  <tabstop>pushButtonSet</tabstop>
  <tabstop>lineEditText</tabstop>
  <tabstop>pushButtonSendText</tabstop>
  <tabstop>lineEditHex</tabstop>
  <tabstop>pushButtonSendHex</tabstop>
  <tabstop>pushButtonCustom</tabstop>
  <tabstop>textEditLog</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
